open_spiel/games/dark_chess_test.cc

```diff
@@ -27,10 +27,20 @@ void BasicDarkChessTests() {
   testing::RandomSimTest(*LoadGame("dark_chess"), 100);
 }
 
+void SmallBoardDarkChessTests() {
```

Comment:
You can merge the two tests together and make it a for loop through possible board sizes.

Reply:
?